src/registry/timefield/ValueDisplay.tsx

```diff
@@ -0,0 +1,60 @@
+import type {TimeComponentSchema} from '@open-formulieren/types';
+import {OrderedList, OrderedListItem} from '@utrecht/component-library-react';
+import {FormattedTime} from 'react-intl';
+
+import './ValueDisplay.scss';
+
+export interface ValueDisplayProps {
+  componentDefinition: TimeComponentSchema;
+  value: string | string[] | undefined;
+}
+
+const formatTime = (timeValue: string): React.ReactNode => {
+  if (!timeValue) return '-';
+
+  // FormattedTime expects a Date object
+  let time = new Date(timeValue);
+
+  if (isNaN(Number(time))) {
+    const [hours, minutes, seconds] = timeValue.split(':');
+    time = new Date();
+    time.setHours(parseInt(hours));
+    time.setMinutes(parseInt(minutes));
+    time.setSeconds(parseInt(seconds));
+  }
+
+  return (
+    <FormattedTime
+      value={time}
+      format="hh:mm:ss"
```

Comment:
I played around a bit with the various options - I think the single prop `timeStyle="short"` instead of format/hour/minute/second would be sufficient and cover localization in multiple languages (it also omits the seconds).

However, it conflicts with https://onzetaal.nl/taalloket/tijden-noteren - the time is reported as `04:15` for example instead of `4.15 uur` which we'd prefer.

I'm not sure if we can do anything with the `format` string here, but I'm tempted to stick with `timeStyle="short"` for now and revisit later.